websites/L/Learnjavascript/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "293803241110503426",
+		"name": "weemizo#7940"
```

The discriminator in the name is a bug, remove.
```suggestion
		"name": "weemizo"
```